### PR TITLE
06 09 2023  as blob storage add domains to zscaler url category

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Change(s):
   - New playbook

   Reason for Change(s):
   - New playbook

   Testing Completed:
   - Yes

image

### CODE REVIEW

**Comment by manishkumar1991** [comment]
@AcceleryntSecurityDev 

After deploying the playbook ,

Condition value is coming empty , 

kindly check

image

File: Playbooks/AS-Blob-Storage-Add-Domains-to-Zscaler-URL-Category/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,888 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "AS-Blob-Storage-Add-Domains-to-Zscaler-URL-Category", 
+        "description": "This playbook will use Azure blob storage to maintain a Zscaler custom URL category of your choice. If the azure blob storage is modified, the Zscaler URL category values will be updated to match.",
+        "prerequisites": "1. A configured Zscaler admin account is needed, along with a username, password, and API token for this account. 2. A Zscaler URL Category is also needed. 3. A Microsoft Azure integration account and Azure key vault containing the Zscaler account password and API token must also be set up. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Blob-Storage-Add-Domains-to-Zscaler-URL-Category",
+        "lastUpdateTime": "2023-06-08T12:12:05Z",
```
Comment (manishkumar1991) [inline Playbooks/AS-Blob-Storage-Add-Domains-to-Zscaler-URL-Category/azuredeploy.json:8]: Missing post deployment steps

**Comment by AcceleryntSecurityDev** [comment]
@manishkumar1991 Thanks, I will address these comments next week

**Comment by AcceleryntSecurityDev** [comment]
> @AcceleryntSecurityDev
> 
> After deploying the playbook ,
> 
> Condition value is coming empty ,
> 
> kindly check
> 
> image

There are additional post deployment steps that address this in the documentation. I was unable to find away to deploy the playbook with the "Get Blob Metadata" component successfully. Since there are environment specific requirements for this component that cannot be left blank or intuitively templatized- I deemed it easier to have the user add this component and a few dependencies post deployment. I will be sure to add this in the "postDeployment" json. 
image
image

**Comment by AcceleryntSecurityDev** [comment]
@manishkumar1991 postDeployment items have been added.

**Comment by AcceleryntSecurityDev** [comment]
@manishkumar1991 following up, have you had a chance to review?

**Comment by manishkumar1991** [comment]
Playbook and readme file looks good

**Review by manishkumar1991 — APPROVED** [review]
playbook approved

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**